android/anc/src/main/java/org/smartregister/fhircore/anc/ui/otp/OtpLoginScreen.kt

```diff
@@ -0,0 +1,265 @@
+          ) { Text(text = stringResource(id = R.string.otp_menu_login)) }
+        }
+      }
+    )
+
+    if (showForgotPinDialog) {
+      ForgotPinDialog(forgotPin = forgotPin, onDismissDialog = { showForgotPinDialog = false })
+    }
+
+    Column(
+      modifier =
+        Modifier.fillMaxSize()
+          .padding(horizontal = 16.dp, vertical = 70.dp)
+          .wrapContentWidth(Alignment.CenterHorizontally)
+    ) {
+      Image(
+        painter = painterResource(id = R.drawable.ic_liberia),
+        contentDescription = stringResource(id = R.string.app_logo),
+        modifier =
+          modifier
+            .padding(top = 16.dp)
+            .align(Alignment.CenterHorizontally)
+            .requiredHeight(120.dp)
+            .requiredWidth(140.dp)
+            .testTag(APP_LOGO_TAG),
+      )
+      Text(
+        text = stringResource(R.string.app_name_ecbis),
+        textAlign = TextAlign.Center,
+        fontWeight = FontWeight.Bold,
+        fontSize = 22.sp,
+        modifier = modifier.padding(top = 16.dp).align(Alignment.CenterHorizontally)
+      )
+
+      Text(
+        text = stringResource(R.string.enter_pin_w4vv01),
+        textAlign = TextAlign.Center,
+        fontWeight = FontWeight.Normal,
+        fontSize = 20.sp,
+        modifier = modifier.padding(vertical = 16.dp).align(Alignment.CenterHorizontally)
+      )
+
+      PinView(
+        otpInputLength = 4,
+        isDotted = true,
+        onPinChanged = onPinChanged,
+        showError = showError
+      )
+
+      if (showError)
+        Text(
+          text = stringResource(R.string.incorrect_pin_please_retry),
+          textAlign = TextAlign.Center,
+          fontWeight = FontWeight.Normal,
+          fontSize = 16.sp,
+          color = colorResource(id = R.color.colorError),
+          modifier = modifier.padding(vertical = 16.dp).align(Alignment.CenterHorizontally)
+        )
+
+      Text(
+        text = stringResource(R.string.forgot_pin),
+        color = LoginButtonColor,
+        fontSize = 16.sp,
+        style = TextStyle(textDecoration = TextDecoration.Underline, color = LoginDarkColor),
+        modifier =
+          modifier.padding(top = 24.dp).align(Alignment.CenterHorizontally).clickable {
+            showForgotPinDialog = !showForgotPinDialog
+          }
+      )
+    }
+  }
+}
+
+@Composable
+fun ForgotPinDialog(
+  forgotPin: () -> Unit,
+  onDismissDialog: () -> Unit,
+  modifier: Modifier = Modifier
+) {
+  AlertDialog(
+    modifier = Modifier.testTag(FORGOT_PIN),
+    onDismissRequest = onDismissDialog,
+    title = {
+      Text(
+        text = stringResource(org.smartregister.fhircore.engine.R.string.forgot_password_title),
+        fontWeight = FontWeight.Bold,
+        fontSize = 18.sp
+      )
+    },
+    text = {
+      Text(
+        text =
+          stringResource(
+            org.smartregister.fhircore.engine.R.string.call_supervisor,
+            "012-3456-789"
```

Comment:
Make this configurable. This will enable the users to update phone numbers of the specific supervisors